fh-mbaas-template-1node-persistent.json

```diff
@@ -38,6 +38,18 @@
       }
     },
     {
+      "kind": "ConfigMap",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "node-proxy"
+      },
+      "data": {
+        "https-proxy" : "${PROXY_URL}",
+        "http-proxy": "${PROXY_URL}",
+        "no-proxy": "fh-ngui, fh-appstore, rhmap-proxy, millicore, fh-scm, fh-supercore, gitlab-shell, fh-messaging, fh-metrics, fh-aaa, ups, mysql, redis, memcached ,nagios, mongodb-1"
```

Comment:
Given that this is in the MBaaS, and not the RHMAP core, the components here should be the MBaaS ones, not the core ones.